src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
Yes, that's why I picked that name when adding support for it. The two intentionally coincide. Why do we need to keep these two separate?